streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -289,6 +289,16 @@ public void testCannotCleanupWhileRunning() throws Exception {
         }
     }
 
+    @Test
+    public void testToString()
+    {
+        streams.start();
+        String streamString = streams.toString();
+        streams.close();
+        Assert.assertNotEquals("streamString should not be empty", "", streamString);
+        Assert.assertNotNull("streamString should not be null", streamString);
```

Comment:
Maybe add an assert checking `streamsString` contains the app-id.   But considering this method will be removed, I'm also fine with it as is.